### ISSUE TITLE
[BUG] TC-IDM-6.2 step 13 and 14 getting INVALID_ACTION

### ISSUE THREAD

**Issue by pankore** [project-chip/connectedhomeip#29203]
### Reproduction steps

After applying the commit https://github.com/project-chip/connectedhomeip/commit/5deb59b6caec86fb40726f2574d4bc2d5423106f

In Step 13 and 14, I am still getting INVALID_ACTION after ACL command.

Tested with v1.2 TE2 all-cluster-app:
[TC-IDM-6.2-step13-14.txt](https://github.com/project-chip/connectedhomeip/files/12594385/TC-IDM-6.2-step13-14.txt)

### Bug prevalence

always 

### GitHub hash of the SDK that was being used

5b4f8004662d00bdb111367fec7d3ea978c23372

### Platform

ameba, other

**Comment by xshuqun** [comment]
@bzbarsky-apple From the test plan of step 13: "ACL command giving only access for ACL cluster, So except ACL cluster command if try to send any other command will get status as unsupported access."
So is there a need to change the test plan for step 13? to getting status as INVALID ACCESS?
The outcome should be invalid access or unsupported access?

**Comment by bzbarsky-apple** [comment]
The test plan is wrong, and it's wrong because it's not doing a _command_.  It's doing a subscribe.  For a subscribe, the right thing here per spec is INVALID_ACTION.

**Comment by cjandhyala** [comment]
@bzbarsky-apple the spec says 'If **both** AttributeRequests and EventRequests are empty' then return INVALID_ACTION' but the scenario in the test plan is 'EventRequests set to path which requires a privilege that is not granted for the cluster in the path'

> If the path is a concrete path: 
> Else if reading the event in the path requires a privilege that is not granted to access the cluster in the path, an [EventStatusIB](https://github.com/project-chip/connectedhomeip/issues/29203#_eventstatusib) SHALL be generated with the UNSUPPORTED_ACCESS Status Code.

**Comment by bzbarsky-apple** [comment]
@cjandhyala Please read what I quoted above.  Just to re-emphasize:
> * ***If no error free existent paths remain, then EventRequests are considered empty.***

**Comment by Apollon77** [comment]
I completely agree here and also summarized that in https://github.com/project-chip/connectedhomeip/issues/34015
